system_tests/lewis_emulators/Rzbrp100/interfaces/stream_interface.py

```diff
@@ -0,0 +1,131 @@
+from lewis.adapters.stream import StreamInterface, Cmd
+from lewis.utils.command_builder import CmdBuilder
+from lewis.core.logging import has_log
+from lewis.utils.replies import conditional_reply
+
+
+@has_log
+class Rzbrp100StreamInterface(StreamInterface):
+
+    in_terminator = "\r\n"
+    out_terminator = "\r\n"
+
+    commands = {
+        
+        # Catch-all command for debugging
+        CmdBuilder("catch_all").arg("^#9.*$").eos().build(),
```

Comment:
This will hide errors - should be removed.

Reply:
Have removed requested lines.